app/screens/home/search/search.tsx

```diff
@@ -140,6 +143,7 @@ const SearchScreen = ({teamId}: Props) => {
     }, [resetToInitial]);
 
     const handleTextChange = useCallback((newValue: string) => {
+        searchRef.current?.focus?.();
```

Comment:
How often does this happen? On any change, or only when we click the options?

On any change may be a bit overkill, but not sure if it has any real effect.

Reply:
I added a `handleModiferTextChange` callback that sets the focus.  I'm not positive the constant setting of focus on every change had a real effect, but I agree that it is not necessary.